### PR TITLE
feat: Create spacy notebook example

### PR DESCRIPTION
**Pull request by mallorih**
This PR create an example notebook for using Unstructured with Spacy.

**Not sure why there are 8 files on this PR. Github automatically added them. I do not have these changes locally**

### CODE REVIEW

**Review by MthwRobinson — CHANGES_REQUESTED** [review]
Notebook code looks good, just made one suggestion to help target the data extraction use case. Couple other notes:

- On those other files that got into the PR, I'd give `git checkout main -- requirements/ ** git checkout main -- docs/requirements.txt` a try, that should revert those files.
- Could you add a brief `README` in the `examples/spacy` folder and also a `requirements.txt` with `spacy` and `unstructured`?
- I'd add a little more explanation for what's going on in the spacy section of the notebook.

**Review by MthwRobinson — APPROVED** [review]
Notebook updates look good! LGTM once you add the `requirements.txt` for the example notebook.

**Comment by mallorih** [comment]
> Notebook updates look good! LGTM once you add the `requirements.txt` for the example notebook.

Forgot to add them to the commit 🤦‍♀️.

File: test_unstructured_ingest/test-ingest-against-api.sh [hunk 1]
```diff
@@ -18,6 +18,6 @@ set +e
 
 if [ "$(find 'api-ingest-output' -type f -printf '.' | wc -c)" != 4 ]; then
```
Comment (MthwRobinson) [inline test_unstructured_ingest/test-ingest-against-api.sh:19]: I think you need `5` here, and then change the `echo` to `5` as well

**Pull request merged**